components/sidebar-navigation/Components/LinkItem.jsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import { cnb } from "cnbuilder";
+
+/**
+ * Link Item for Sidebar Navigation
+ *
+ * @returns {JSX.Element}
+ * @constructor
+ */
+export default function LinkItem({ level, url, shortName, active }) {
+  return (
+    <a
```

Comment:
I tested with a menu label that spans 2 lines. Looks like it might be good to add the line height to tighten it up a bit 😄 It says in the mockup is 125% line height, but if you want to go with existing TW utilities you could try su-leading-display (1.2) which is close (or go with su-leading-[125%])

Current:
![Screenshot 2024-07-25 at 2 40 26 PM](https://github.com/user-attachments/assets/017ecaa2-f432-4234-9ee7-f9f4aefa9a62)

120% line-height:
![Screenshot 2024-07-25 at 2 40 18 PM](https://github.com/user-attachments/assets/040f9d34-b0c5-46e4-ab12-25f0a2c4aac7)